src/components/app/details/triggerView/BranchRegexModal.tsx

```diff
@@ -0,0 +1,160 @@
+                                            <span className="mr-14">
+                                                {mat.gitMaterialUrl.includes('gitlab') ? <GitLab /> : null}
+                                                {mat.gitMaterialUrl.includes('github') ? <GitHub /> : null}
+                                                {mat.gitMaterialUrl.includes('bitbucket') ? <BitBucket /> : null}
+                                                {mat.gitMaterialUrl.includes('gitlab') ||
+                                                mat.gitMaterialUrl.includes('github') ||
+                                                mat.gitMaterialUrl.includes('bitbucket') ? null : (
+                                                    <Git />
+                                                )}
+                                            </span>
+                                            <span>
+                                                <div className="fw-6 fs-14">{mat.gitMaterialName}</div>
+                                                <div className="pb-12">
+                                                    Use branch name matching&nbsp;
+                                                    <span className="fw-6">
+                                                        {getBranchRegexName(mat.gitMaterialId) || mat.regex}
+                                                    </span>
+                                                </div>
+                                            </span>
+                                        </div>
+                                        <input
+                                            tabIndex={1}
+                                            placeholder="Enter branch name matching regex"
+                                            className="form__input ml-36 w-95"
+                                            name="name"
+                                            value={_regexValue?.value}
+                                            onChange={(e) =>
+                                                handleRegexInputValue(mat.gitMaterialId, e.target.value, mat)
+                                            }
+                                            autoFocus
+                                            autoComplete="off"
+                                        />
+                                        {_regexValue?.value &&
+                                            _regexValue.isInvalid &&
+                                            renderValidationErrorLabel('Branch name does not match the regex.')}
+                                        {!_regexValue?.value && renderValidationErrorLabel('This is a required field')}
+                                    </div>
+                                )
+                            )
+                        })}
+                </div>
+                {showWebhookModal ? null : renderMaterialRegexFooterNextButton(context)}
```

Comment:
Can we change this to {!showWebhookModal  && renderMaterialRegexFooterNextButton(context)}